data/json/overmap_specials.json

```diff
@@ -1481,5 +1481,47 @@
         "occurrences" : [0, 5],
         "rotate" : true,
         "flags" : [ "CLASSIC" ]
+    },{
+        "type" : "overmap_special",
+        "id" : "rest_stop",
+        "overmaps" :
+        [
+            { "point":[0,0,0], "overmap": "roadstop_north", "connect":"road"}
+        ],
+        "locations" : [ "by_hiway" ],
+        "city_distance" : [10, 200],
+        "city_sizes" : [1, 12],
+        "occurrences" : [ 1,2 ],
```

Comment:
I dont think its a good idea to  force the roadstops' spawn, since this will make the other by_hiway locations much rarer.